packages/volto-slate/src/blocks/Table/deconstruct.js

```diff
@@ -19,7 +19,7 @@ import {
 export function syncCreateTableBlock(rows) {
   const id = uuid();
   const block = {
-    '@type': 'table',
+    '@type': 'slateTable',
```

Comment:
I have no idea why this wasn't flagged before.

Reply:
ups